src/linkPanel/widgets/identityLinks.tsx

```diff
@@ -0,0 +1,38 @@
+import { Button, deleteIcon, LabIcon } from '@jupyterlab/ui-components';
+import React from 'react';
+import { IComponentLinkInfo } from '../types';
+
+/**
+ * A React widget with the identity links.
+ *
+ * @param links - List of links to display.
+ * @param clickCallback - Function to call when clicking on the delete icon.
+ */
+export function identityLinks(
+  links: [IComponentLinkInfo, boolean][],
+  clickCallback: (link: IComponentLinkInfo) => void
+): JSX.Element {
+  return (
+    <table>
+      {links.map(link => (
+        <tr>
```

Comment:
Do we need the key props for the `tr` element?

Reply:
Thanks @trungleduc 
I added it, but strangely react was not complaining about it.